src/test/java/pages/BasePage.java

```diff
@@ -3,23 +3,32 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Parameters;
 
 import java.time.Duration;
 
 public class BasePage {
-    WebDriver driver;
-    WebDriverWait wait;
+    public static WebDriver driver;
+    public static WebDriverWait wait;
     Actions actions;
-    By userAvatarIcon = By.cssSelector("img.avatar");
-
     public BasePage(WebDriver givenDriver) {
         driver = givenDriver;
         wait = new WebDriverWait(driver, Duration.ofSeconds(5));
         actions = new Actions(driver);
     }
+    @BeforeMethod
+    @Parameters({"baseURL"})
+    public static void setUpBrowser(String baseURL) {
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        driver.manage().window().maximize();
+        driver.get(baseURL);
```

Comment:
Please move this method in BaseTest

**Base Test** in the main class which will take care of Browser setup, loading configuration file and other reusable methods like screenshot, handling sync issues and many more.

![image](https://user-images.githubusercontent.com/104761482/212842541-0636ddda-8f8b-42ee-a427-1e2ff227fa23.png)